README.md

```diff
@@ -1,11 +1,52 @@
-# solution-pack-soc-simulator
+# SOC Simulator Solution Pack
+
+## Overview
+
+### Introduction
 
 The FortiSOAR™ SOC Simulator Solution Pack (solution-pack-soc-simulator). This solution pack enables users to experience the power and capability of FortiSOAR™ incident response for SOC Simulator scenarios. This pack contains simulation data and utilities for demonstrating FortiSOAR capabilities around a variety of important SOC use-cases without having to integrate with actual device endpoints.
```

Comment:
The FortiSOAR™ SOC Simulator Solution Pack (solution-pack-soc-simulator). This solution pack enables users ---> The FortiSOAR™ SOC Simulator Solution Pack (solution-pack-soc-simulator) enables users

Reply:
This pack contains simulation data --> IMO, this pack doesn't contain any sim data. Please confirm

Reply:
Need more details her...

Reply:
This is a framework SP for all Simulation scenarios. We should document it like framework and its capabilities.

Reply:
> The FortiSOAR™ SOC Simulator Solution Pack (solution-pack-soc-simulator). This solution pack enables users ---> The FortiSOAR™ SOC Simulator Solution Pack (solution-pack-soc-simulator) enables users

Done

Reply:
> This pack contains simulation data --> IMO, this pack doesn't contain any sim data. Please confirm

Removed this line, please check